src/psyclone/tests/config_test.py

```diff
@@ -607,6 +621,42 @@ def test_mappings():
     assert "Invalid format for mapping: k2=v2" in str(err.value)
 
 
+def test_numeric_mappings():
+    '''Test the definition of a mapping in the config file.'''
+    mapping = APISpecificConfig.create_numeric_dict_from_list(
+        ["k1:1", "k2:2"]
+        )
+    assert mapping == {"k1": 1, "k2": 2}
+
+    mapping = APISpecificConfig.create_numeric_dict_from_list([])
+    assert not mapping
+
+    # Check float format string values are converted to int
```

Comment:
Unless this float handling is needed somewhere (I haven't spotted any use of it so far) then, as suggested in the code, I'd remove this.